[2023-04-14 17:19:02,884] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:19:01.478020+00:00 [queued]>
[2023-04-14 17:19:02,892] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:19:01.478020+00:00 [queued]>
[2023-04-14 17:19:02,893] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:19:02,894] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 17:19:02,895] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:19:02,905] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-14 17:19:01.478020+00:00
[2023-04-14 17:19:02,912] {standard_task_runner.py:52} INFO - Started process 122 to run task
[2023-04-14 17:19:02,916] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'manual__2023-04-14T17:19:01.478020+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmpa2he8vrj', '--error-file', '/tmp/tmp0nhjbl1g']
[2023-04-14 17:19:02,919] {standard_task_runner.py:80} INFO - Job 133: Subtask get_cv_lac_data
[2023-04-14 17:19:02,989] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:19:01.478020+00:00 [running]> on host 9a187b96ec59
[2023-04-14 17:19:03,057] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T17:19:01.478020+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T17:19:01.478020+00:00
[2023-04-14 17:19:03,059] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-14 17:19:03,060] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 17:19:03,072] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T171901, start_date=20230414T171902, end_date=20230414T171903
[2023-04-14 17:19:03,129] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 17:19:03,160] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
